### ISSUE TITLE
pyqt4 depends on python-sip

### ISSUE THREAD

**Issue by titobrasolin** [mkschreder/openwrt-xorg-feed#3]
@mkschreder I just posted a pull request for inclusion of python-sip into openwrt/packages: https://github.com/openwrt/packages/pull/2658

If it's accepted I'll update the current issue and submit an update for pyqt4

**Comment by mkschreder** [comment]
What package is that? I see only one reference to it as host dependency for building pyqt4. Is that the only place where it is needed?

**Comment by mkschreder** [comment]
Is it this one: https://dev.openwrt.org/browser/packages/lang/python-sip/Makefile?rev=22715

**Comment by mkschreder** [comment]
How about simply moving it to this feed if pyqt4 is the only package that uses it today?

**Comment by titobrasolin** [comment]
I agree: this feed would be a nice new home for python-sip, I'll prepare a pull request soon.
By the way, it's currently in "oldpackages" http://git.openwrt.org/packages.git

**Comment by titobrasolin** [comment]
Hi Martin, there is still no news from openwrt/packages. In the meanwhile, I started using your feed in my build script: https://github.com/titobrasolin/openwrt/blob/master/build.sh

#4 and #5 will be my last pull requests, at least for awhile: thank you so much for your feedback and collaboration.
